src/scribe_data/utils.py

```diff
@@ -240,6 +244,85 @@ def get_language_words_to_ignore(language: str) -> list[str]:
     )
 
 
+def _get_language_dir_path(language):
+    """
+    Constructs the directory path for a given language's data within the project.
+
+    Parameters
+    ----------
+        language : str
+            The name of the language for which the directory path is needed.
+
+    Returns
+    -------
+        str
+            The directory path for the specified language's data.
+    """
+    PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+    return f"{PATH_TO_SCRIBE_ORG}/Scribe-Data/src/scribe_data/extract_transform/languages/{language}"
+
+
+def load_queried_data(file_path, language, data_type):
+    """
+    Loads queried data from a JSON file for a specific language and data type.
+
+    Parameters
+    ----------
+        file_path : str
+            The path to the file containing the queried data.
+        language : str
+            The language for which the data is being loaded.
+        data_type : str
+            The type of data being loaded (e.g., 'nouns', 'verbs').
+
+    Returns
+    -------
+        tuple
+            A tuple containing the loaded data, a boolean indicating whether the data is in use,
+            and the path to the data file.
+    """
+    queried_data_file = f"{data_type}_queried.json"
+    update_data_in_use = False
+
+    if f"languages/{language}/{data_type}/" not in file_path:
+        data_path = queried_data_file
+    else:
+        update_data_in_use = True
+        data_path = f"{_get_language_dir_path(language)}/{data_type}/{queried_data_file}"
+
+    with open(data_path, encoding="utf-8") as f:
+        return json.load(f), update_data_in_use, data_path
+
+
+def export_formatted_data(formatted_data, update_data_in_use, language, data_type):
+    """
+    Exports formatted data to a JSON file for a specific language and data type.
+
+    Parameters
+    ----------
+        formatted_data : dict
+            The data to be exported.
+        update_data_in_use : bool
+            A flag indicating whether the data is currently in use.
+        language : str
+            The language for which the data is being exported.
+        data_type : str
+            The type of data being exported (e.g., 'nouns', 'verbs').
+
+    Returns
+    -------
+        None
+    """
+    if update_data_in_use:
+        export_path = f"{_get_language_dir_path(language)}/formatted_data/{data_type}.json"
+    else:
+        export_path = f"{data_type}.json"
+
+    with open(export_path, "w", encoding="utf-8") as file:
+        json.dump(formatted_data, file, ensure_ascii=False, indent=0)
+    print(f"Wrote file {data_type}.json with {len(formatted_data):,} {data_type}.")
+
+
```

Comment:
> .. Also, let me know if tests should be added for these functions?

Honestly, tests would be amazing if you'd like to add those for these new functions as well :smile: 

Though, totally not needed for this PR. Haven't been strict on always adding tests for everything new, and this PR is already pretty substantial as is. So definitely feel free to leave it for later if you would like :wink: - especially considering that tests for two of the functions here likely involve validating that correct JSON is getting read/written by them.